examples/02_fit_predict_plot_employee_salaries.py

```diff
@@ -1,17 +1,14 @@
 """
-Predicting the salary of employees
-==================================
-
-The `employee salaries <https://catalog.data.gov/dataset/employee-salaries-2016>`_
-dataset contains information
-about annual salaries (year 2016) for more than 9,000 employees of the 
-Montgomery County (Maryland, US). In this example, we are interested
-in predicting the column *Current Annual Salary*
-depending on a mix of clean columns and a dirty column.
-We choose to benchmark different categorical encodings for
-the dirty column *Employee Position Title*, that contains
-dirty categorical data.
+Comparing encoders of a dirty categorical columns
+==================================================
 
+Considering a database on `employee salaries
+<https://catalog.data.gov/dataset/employee-salaries-2016>`_, one problem
+is that the *Employee Position Title* column contains dirty categorical
+data.
```

Comment:
```suggestion
The column *Employee Position Title* of the dataset `employee salaries
<https://catalog.data.gov/dataset/employee-salaries-2016>`_ contains dirty categorical
data.
```

That's how I would rephrase it.
Also, the link is dead.